Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -25,9 +32,11 @@ final class PledgeDescriptionCell: UITableViewCell, ValueCell {
   private lazy var estimatedDeliveryLabel: UILabel = { UILabel(frame: .zero) }()
   private lazy var dateLabel: UILabel = { UILabel(frame: .zero) }()
   private lazy var descriptionLabel: UILabel = { UILabel(frame: .zero) }()
-  private lazy var learnMoreLabel: UILabel = { UILabel(frame: .zero) }()
-  private lazy var spacerView: UIView = {
-    return UIView(frame: .zero) |> \.translatesAutoresizingMaskIntoConstraints .~ false }()
+  private lazy var spacerView: UIView = { UIView(frame: .zero) }()
```

Comment:
Do we need the closure here? Or can't we just do `private lazy var spacerView = UIView(frame: .zero)`?

Reply:
Good point! We actually don't need the closure. So there's going to be a number of places that we can change this. I think the function stuff is a hangover from having set local vars in that scope before to set properties on it, but seeing as we pipe forward to set these things it can all be done in-line.